docker/docker-compose.yaml

```
@@ -0,0 +1,68 @@
+version: '3'
+
+services:
+  front_end_container:
+    build:
+      context: .
+      dockerfile: SustainML.Dockerfile
+    container_name: front_end_container
+    environment:
+      - node=orchestrator
+      - DISPLAY=${DISPLAY}
+    volumes:
+      - /tmp/.X11-unix:/tmp/.X11-unix
+    network_mode: "host"
```

For each of the services, and it works for me !
```suggestion
    network_mode: host
    ipc: host
```